docs/operations.md

```diff
@@ -123,6 +242,53 @@ stays in the database:
 
 **Default**: `30` (days)
 
+### Always keep scavenged (_Deprecation Notice_)
+
+This option ensures that the newer chunk from a scavenge operation is always kept.
+
+| Format               | Syntax                             |
+|:---------------------|:-----------------------------------|
+| Command line         | `--always-keep-scavenged`          |
+| YAML                 | `AlwaysKeepScavenged`              |
+| Environment variable | `EVENTSTORE_ALWAYS_KEEP_SCAVENGED` |
+
+### Scavenge backend page size
+
+Specify the page size of the scavenge database. The default value is 16KiB.
+
+| Format               | Syntax                                  |
+|:---------------------|:----------------------------------------|
+| Command line         | `--scavenge-backend-page-size`          |
+| YAML                 | `ScavengeBackendPageSize`               |
+| Environment variable | `EVENTSTORE_SCAVENGE_BACKEND_PAGE_SIZE` | 
+
+### Scavenge backend cache size
+
+Specify the amount of memory, in bytes, to use for backend caching during scavenging. The default value is 64MiB.
+
+| Format               | Syntax                                   |
+|:---------------------|:-----------------------------------------|
+| Command line         | `--scavenge-backend-cache-size`          |
+| YAML                 | `ScavengeBackendCacheSize`               |
+| Environment variable | `EVENTSTORE_SCAVENGE_BACKEND_CACHE_SIZE` |                       
+
+### Scavenge hash users cache capacity
+
+Specify the number of stream hashes to remember when checking for collisions.
+
+| Format               | Syntax                                          |
+|:---------------------|:------------------------------------------------|
+| Command line         | `--scavenge-hash-users-cache-capacity`          |
+| YAML                 | `ScavengeHashUsersCacheCapacity`                |
+| Environment variable | `EVENTSTORE_SCAVENGE_HASH_USERS_CACHE_CAPACITY` |
+
+### Best Practices
```

Comment:
This is a great set of best practices. They're in the 'options' section at the moment though and should probably be somewhere else.

Some of the best practices are also already written elsewhere in this same document but I think the wording here is better so we can remove the old